service/src/main/java/bio/terra/catalog/rawls/RawlsService.java

```diff
@@ -81,6 +85,76 @@ WorkspacesApi workspacesApi(AuthenticatedUserRequest user) {
     return new WorkspacesApi(getApiClient(user));
   }
 
+  //  public List<Entity> entityList(AuthenticatedUserRequest user, String storageSourceId)  {
+  //    WorkspaceResponse response = null;
+  //    try {
+  //      response = workspacesApi(user).getWorkspaceById(storageSourceId, List.of());
+  //      var entities = entitiesApi(user).listEntities(response.getWorkspace().getNamespace(),
+  // response.getWorkspace().getName(), Sample);
+  //      return entities;
+  //    } catch (ApiException e) {
+  //      throw new RuntimeException(e);
+  //    }
+  //  }
+
+  public EntityQueryResponse entityQuery(
+      AuthenticatedUserRequest user, String storageSourceId, String tableName) {
```

Comment:
`storageSourceId` is the name of this concept in the caller. In this code, it's `workspaceId`.
```suggestion
      AuthenticatedUserRequest user, String workspaceId, String tableName) {
```